requests_mv_integrations/request_mv_integration.py

```diff
@@ -0,0 +1,1774 @@
+            'allow_redirects': allow_redirects,
+            'verify': verify,
+            'stream': stream,
+            'verbose': verbose
+        }
+
+        time_start_req = dt.datetime.now()
+
+        if not request_retry_http_status_codes:
+            request_retry_http_status_codes = REQUEST_RETRY_HTTP_STATUS_CODES
+        self.request_retry_http_status_codes = request_retry_http_status_codes
+
+        if not request_retry_excps:
+            request_retry_excps = REQUEST_RETRY_EXCPS
+        self.request_retry_excps = request_retry_excps
+
+        self.logger.debug(
+            "Request: Details: {}".format(
+                request_label if request_label else ""
+            ),
+            extra={
+                'request_method': request_method,
+                'request_retry': request_retry,
+                'request_url': request_url,
+                'request_params': request_params,
+                'request_data': request_data,
+                'request_json': request_json,
+                'request_headers': request_headers,
+                'request_auth': request_auth,
+                'request_session': request_session,
+                'timeout': timeout,
+                'allow_redirects': allow_redirects,
+                'verify': verify,
+                'stream': stream,
+                'verbose': verbose
+            }
+        )
+
+        try:
+            response = self._request_retry(
+                call_func=self._request,
+                fargs=None,
+                fkwargs=kwargs,
+                timeout=timeout,
+                request_retry_func=request_retry_func,
+                request_retry_excps_func=request_retry_excps_func,
+                retry_tries=retry_tries,
+                retry_delay=retry_delay,
+                retry_max_delay=retry_max_delay,
+                retry_backoff=retry_backoff,
+                retry_jitter=retry_jitter,
+                request_label=request_label
+            )
+
+        except (
+            requests.exceptions.ConnectTimeout,
+            requests.exceptions.ReadTimeout,
+            requests.exceptions.Timeout
+        ) as ex_req_timeout:
+            raise TuneRequestError(
+                error_message="Request: Exception: Timeout",
+                errors=ex_req_timeout,
+                error_request_curl=self.built_request_curl,
+                exit_code=TuneIntegrationExitCode.GATEWAY_TIMEOUT
+            )
+
+        except requests.exceptions.HTTPError as ex_req_http:
+            raise TuneRequestError(
+                error_message="Request: Exception: HTTP Error",
+                errors=ex_req_http,
+                exit_code=TuneIntegrationExitCode.MOD_ERR_REQUEST_HTTP,
+                error_request_curl=self.built_request_curl
+            )
+
+        except requests.exceptions.ConnectionError as ex_req_connect:
+            raise TuneRequestError(
+                error_message="Request: Exception: Connection Error",
```

Comment:
I realized, I am referring to the blocks that use the same exit code. I will comment on those spots